aspnet/2-structured-data/Models/DatastoreBookStore.cs

```diff
@@ -29,6 +35,8 @@ public static class DatastoreBookStoreExtensionMethods
         /// <returns>A datastore key.</returns>
         public static Key ToKey(this long id)
         {
+            // P0 This function is especially useful, but I see no way to implement it

```

Comment:
The only thing I know is the kind.  With the Apiary API, that was sufficient.  Why is it not sufficient with the new API?